kilkari/src/main/java/org/motechproject/nms/kilkari/domain/SubscriptionPack.java

```diff
@@ -13,21 +14,26 @@
 public class SubscriptionPack {
 
     @Field
+    @Unique
     private String name;
 
     @Field
     private SubscriptionPackType type;
```

Comment:
If you know additional constraints on the field please add them (i.e. not null, or column size)